test/test_transforms_v2_refactored.py

```diff
@@ -261,7 +263,133 @@ def _check_transform_v1_compatibility(transform, input, *, rtol, atol):
     _script(v1_transform)(input)
 
 
-def check_transform(transform, input, check_v1_compatibility=True):
+def _make_transform_sample(transform, *, image_or_video, adapter):
+    device = image_or_video.device if isinstance(image_or_video, torch.Tensor) else "cpu"
+    size = F.get_size(image_or_video)
+    input = dict(
+        image_or_video=image_or_video,
+        image_tv_tensor=make_image(size, device=device),
+        video_tv_tensor=make_video(size, device=device),
+        image_pil=make_image_pil(size),
+        bounding_boxes_xyxy=make_bounding_boxes(
+            size, format=tv_tensors.BoundingBoxFormat.XYXY, num_objects=3, device=device
```

Comment:
I know it comes from the original test, but we don't seem to have a good reason to set `num_objects` (previously `batch_dims)`?

Maybe we can avoid it (and avoid introducing the parameter in `make_bounding_boxes`)?